src/routes/(console)/organization-[organization]/+page.svelte

```diff
@@ -129,7 +130,9 @@
     }
 
     function formatName(name: string, limit: number = 19) {
-        return name ? (name.length > limit ? `${name.slice(0, limit)}...` : name) : '-';
+        const mobileLimit = 12;
```

Comment:
I think it should work till `16` when I checked. Can you verify which is the smallest working that doesn't push the Tag as well?